docs/source/models/intervention_models/alzheimers/testing_diagnosis.rst

```diff
@@ -30,4 +30,185 @@
 
 ==========================================
 Alzheimer's Testing and Diagnosis
-==========================================
\ No newline at end of file
+==========================================
+
+Alzheimer's testing will be modeled as an intervention with no affected outcomes
+which introduces PET, CSF, and BBBM testing for eligible pre-dementia populations, 
+as defined in :ref:`Reference Scenario and Alternative Scenario 1 <alz_scenarios>`.
+
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - BBBM
+    - Blood-based biomarker
+    - BBBM tests measure blood plasma protein levels and are less invasive than CSF or PET tests
+  * - CSF
+    - Cerebrospinal Fluid
+    - CSF tests require a lumbar puncture
+  * - PET
+    - Positron Emission Topography
+    - PET scanners measure brain protein levels are only available at specialized medical facilities
+
+
+Intervention Overview
+-----------------------
+
+Alzheimer's testing is classified into two groups - baseline (CSF and PET) testing, 
+and BBBM testing. Baseline testing already exists in present-day conditions, before the 
+hypothetical introduction of BBBM testing in :ref:`Alternative Scenario 1 <alz_scenarios>`.
+The introduction of BBBM testing will replace some number of the baseline tests in 
+Alternative Scenario 1, and will also be used to inform treatment in Alternative 
+Scenario 2. 
+
+Vivarium Modeling Strategy
+--------------------------
+
+Baseline testing - CSF and PET
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CSF and PET are existing biomarker tests which can identify cases of Alzheimer's disease.
+We model CSF and PET tests in our :ref:`Reference Scenario <alz_scenarios>` in order to
+estimate the number of these more expensive, invasive tests which would be replaced
+by the introduction of BBBM tests in :ref:`Alternative Scenario 1 <alz_scenarios>`.
+
+Location-specific testing rates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+US testing rates among patients who are diagnosed with MCI or dementia are given 
+in Figure 3 `here <https://pmc.ncbi.nlm.nih.gov/articles/PMC12321507/>`_ (overall cohort).
```

Comment:
Can you please add a references section? See example: https://vivarium-research.readthedocs.io/en/latest/models/intervention_models/intrapartum/misoprostol_intervention.html#references